packages/design-system/src/index.js

```diff
@@ -23,12 +23,14 @@ import Modal from 'react-modal';
  * Internal dependencies
  */
 import localStore, { LOCAL_STORAGE_PREFIX } from './utils/localStore';
+import sessionStore, { SESSION_STORAGE_PREFIX } from './utils/sessionStore';
 import * as Icons from './icons';
 
 const { setAppElement } = Modal;
 
 export { Icons };
 export { localStore, LOCAL_STORAGE_PREFIX };
+export { sessionStore, SESSION_STORAGE_PREFIX };
```

Comment:
Any particular reason this uses `sessionStorage` rather than `localStorage`? Is it a security concern? Ping @swissspidy

Reply:
Probably because there's no reason for keeping the backup persistently since this is mainly thought for unexpected happenings and becomes irrelevant.

Reply:
It matches WordPress behavior and also prevents leaking data on a shared computer for example